_includes/mobile/intro-prebid-rendered.md

```diff
@@ -0,0 +1,128 @@
+
+### Prerequisites
+
+The GAM Prebid-Rendered Integration method assumes that you have the following components:
+
+- **Google Ad Manager Account** - A GAM account allows you to manage and serve ads within your mobile app. Within this account you'll need to configure your inventory and create orders for serving ads within your app. This involves defining ad units (spaces within your app where ads will be displayed) and setting up orders and line items to deliver ads to those units. See [Prebid's AdOps Guide](/adops/before-you-start.html) for more information.
+- **Google Mobile Ads (GMA) SDK** - This refers to the software development kit provided by Google. You need to ensure that you have the latest version of the GMA SDK supported by Prebid SDK. This SDK integration is necessary to communicate with the ad server and display ads in your app.
+- **Prebid SDK** - You will need the latest version of the Prebid Mobile SDK for either [Android](/prebid-mobile/pbm-api/android/code-integration-android.html) or [iOS](/prebid-mobile/pbm-api/ios/code-integration-ios.html).
+- **Prebid Server** - You will need a cluster of servers running [Prebid Server](/prebid-server/use-cases/pbs-sdk.html). You can set up your own Prebid Server or work with a [Prebid Server managed service](https://prebid.org/managed-services/). Prebid Server provides you with the following:
+  - Configuration storage - rather than hardcoding all the details of your current business arrangements in the app, Prebid Server stores which bidders you're currently working with, their inventory details, and other settings that can be changed without updating your app.
+  - Server-side auction - the server will make the connections to multiple auction bidding partners so the app doesn't have to.
+  - Privacy regulation tools - the server can help your legal team meet different regulatory needs in different jurisdictions by configuring various protocols and anonyimization activities.
+
+## How it Works
+
+Here's how the ad bidding-auction-rendering process works in this integration scenario.
+
+![GAM Prebid-Rendered Integration Details](/assets/images/prebid-mobile/mobile-details-gam-prebid-rendered.png)
+
+1. Prebid SDK calls Prebid Server which supplies one or more bids.
+1. PBSDK adds targeting values to GMA SDK.
+1. GMA SDK calls GAM, which makes the ad decision.
+1. If a 3rd party HTML creative is chosen (banner, native, interstitial, non-instream video):
+    1. GMA SDK writes HTML to a webview, loading google_mobile_app_ads.js.
+    1. This script triggers a "PrebidAppEvent".
+    1. Which in turn triggers PBSDK to handle the rendering of the creative:
+        1. PBSDK fires the Prebid Server win event.
+        1. If a 3rd party rendering SDK is in use, PBSDK calls it for rendering.
+        1. Otherwise, it writes the creative into an iframe and:
```

Comment:
In this integration scenario, Prebid SDK writes the creative into internal WebView and replaces the GAMSDK's WebView on the screen with its own one. 

The key point - the rendering of bids is performed by Prebid SDK not GMA SDK.